etcd.py

```diff
@@ -267,6 +266,22 @@ def etcd_arguments(self, data_dir, initial_cluster, cluster_state):
             cluster_state
         ]
 
+        # this section handles etcd version specific flags
+        etcdversion = os.environ.get('ETCDVERSION')
+        if etcdversion:
+            etcdversion = list(map(lambda x: int(x), etcdversion.split('.')))
```

Comment:
`etcdversion = [int(v) for v in etcdversion.split('.')]`

Reply:
Thanks, I changed it.